src/Bicep.LangServer.IntegrationTests/CompletionTests.cs

```diff
@@ -3565,15 +3565,15 @@ public async Task VerifyCompletionRequestModuleDependsOn_ResourceSymbolsVeryHigh
 }";
 
             var (text, cursors) = ParserHelper.GetFileWithCursors(mainContentWithCursors, '|');
-            Uri mainUri = new("file:///main.bicep");
-            var files = new Dictionary<Uri, string>
+            DocumentUri mainUri = "file:///main.bicep";
+            var files = new Dictionary<DocumentUri, string>
             {
-                [new Uri("file:///mod.bicep")] = moduleContent,
-                [new Uri("file:///mod2.bicep")] = moduleContent,
```

Comment:
There's a lot of duplicate code in these tests that could be simplified, but that's outside the scope of this PR.